multimodal/src/autogluon/multimodal/data/process_rois.py

```diff
@@ -0,0 +1,216 @@
+import logging
```

Comment:
It seems that the corresponding code in `process_image.py` is not removed yet.

Reply:
Yes, but if we remove them we can't load old predictor checkpoints.

Reply:
We can create a new rois processor during loading and replace the saved one. The refactor seem not done with the existence and usage of old code.

Reply:
Nice suggestion! I'll try.

Reply:
I have finished removing the code from process_image. However I find that to reduce backward compatibility codes we'd better write a processor for all mmlab models to avoid future refactors that change the processor's API.